Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -2166,7 +2166,7 @@ bool RenderBox::pushContentsClip(PaintInfo& paintInfo, const LayoutPoint& accumu
     if (paintInfo.phase == PaintPhase::BlockBackground || paintInfo.phase == PaintPhase::SelfOutline || paintInfo.phase == PaintPhase::Mask)
         return false;
 
-    bool isControlClip = hasControlClip();
+    bool isControlClip = paintInfo.phase != PaintPhase::EventRegion && hasControlClip();
```

Comment:
@etiennesegonzac are you ok with this change?

Reply:
it makes sense for Interaction Regions, but we need to look at Event Regions on iOS a bit more:
https://ews-build.s3-us-west-2.amazonaws.com/iOS-17-Simulator-WK2-Tests-EWS/9a2818d6-21213/results.html
(not worried about the text field ones, but the anchor-lock.html needs some investigation!)

Reply:
Do we understand how big things get without the control clip? Are there controls that can have significant overspill that we just don't happen to have tests for? Try removing it for the normal paint code and eyeball the results for all kinds of controls.

Reply:
I already added some tests for select element control overspilling into the new `select-element.html` test file and it has correct respective clipping for both the input and interaction region, but I'll add more tests for other kinds of overspill with other controls.